src/cli.ts

```diff
@@ -66,11 +66,19 @@ class OdoChannelImpl implements OdoChannel {
     }
 
     prettifyJson(str: string) {
+        const tokenRegex = /--token\s*=\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const credentialRegex = /-p\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
         let jsonData: string;
         try {
             jsonData = JSON.stringify(JSON.parse(str), null, 2);
         } catch (ignore) {
-            return str;
+            const hideTokenRegex = str.match(tokenRegex);
+            if (hideTokenRegex) {
+                return hideTokenRegex ? str.replace(hideTokenRegex[1], '**********'): str;
+            } else {
+                const hideCredentialRegex = str.match(credentialRegex);
+                return (hideCredentialRegex) ? str.replace(hideCredentialRegex[0], '-p **********'): str;
```

Comment:
This method could use the same treatment as the openshiftItem ones. Please see the comments about these.

Reply:
Why are you not reusing  the methods from openShiftItem ?